### PR TITLE
Remove nfs dependency for local development

### PR DESCRIPTION
**Pull request by spaceo**
#### Link to issue

https://reload.atlassian.net/browse/DDFSOEG-348

#### Description

We have recently experienced issues that seem to have occurred between OSX Ventura, NFS and Yarn.
When we run yarn install as part of ci task
- Lighthouse, Pa11y or Cypress on a "fresh setup"/without cache, the command stalls and we never get to fetch packages.

If we omit the NFS mount and instead use VirtioFS, which has been added in newer versions of Docker for Mac, the problem does not arise.

#### Checklist

- [x] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [x] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [x] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

If you have any further comments or questions for the reviewer them please add them here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,10 +21,12 @@ In order to run local development you need:
...
 ## Other initial steps
 
-If you are using a mac/OSX it is recommended to use nfs on the mounted volumes
+If you are using a mac/OSX it is recommended to use [VirtioFS](https://virtio-fs.gitlab.io) on the mounted volumes
 in docker-compose.
 
-Look at [mac-nfs.readme.md](mac-nfs.readme.md) in order to set it up.
+In your Docker for mac preference you need to activate VirtioFS for directory sharing:
+
+![OSX preference pane providing access to NFSD](documentation/images/virtiofs.png)
```
Comment (kasperg) [inline README.md:29]:
```suggestion
![OSX preference pane providing access to VirtioFS](documentation/images/virtiofs.png)
```

File: mac-nfs.readme.md [hunk 2]
```diff
@@ -1,33 +0,0 @@
...
-Assuming you're keeping your Docker/code projects in `~/code`, you'll want to add
-this line to your `/etc/exports`:
-
-`/System/Volumes/Data/Users/<USERNAME>/code -mapall=<USERNAME>:staff -alldirs localhost`
-
-## Full access
-
-`/sbin/nfsd` and your terminal program needs "Full disk access":
-
-![OSX preference pane providing access to NFSD](documentation/images/nftd.png)
```
Comment (kasperg) [inline mac-nfs.readme.md:19]: Please remember to delete the image at `documentation/images/nftd.png`.

**Review by kasperg — APPROVED** [review]
:+1: LGTM. A few changes for you to consider before you merge.

File: mac-nfs.readme.md (hunk 2 above)
Comment (spaceo) [reply mac-nfs.readme.md:19]: Tes 👍 Done

**Review by cableman — APPROVED** [review]

**Pull request merged**